docs/specification/README.md

```diff
@@ -166,7 +166,7 @@ Channel Info records are uniquely identified within a file by their channel ID.
 | 2 | id | uint16 | A unique identifier for this channel within the file. |
 | 4 + N | topic | String | The channel topic. |
 | 4 + N | message_encoding | String | Encoding for messages on this channel. The value should be one of the [well-known message encodings](./well-known-encodings.md). Custom values should use `x-` prefix. |
```

Comment:
Keeping this as `ChannelInfo.message_encoding` rather than `ChannelInfo.encoding` because it is a property of the message, not the channel (and to avoid confusion with `Schema.encoding`).